spec/registry.md

```diff
@@ -605,6 +630,14 @@ MUST be multiplied by 100 for the purposes of formatting.
 
 The _function_ `:integer` performs selection as described in [Number Selection](#number-selection) below.
 
+#### Composition
+
+When an _operand_ or an _option_ value uses a _variable_ annotated,
```

Comment:
"uses" seems wrong to me?

```suggestion
When an _operand_ or the value of an _option_ consists of a _variable_ previously annotated,
```

Reply:
This language was suggested by @stasm in https://github.com/unicode-org/message-format-wg/pull/814#discussion_r1668948100, and is currently the same across the three PRs. Would it be ok to not nitpick on this now, and iterate on the exact language once these changes have first landed?